core/src/global.rs

```diff
@@ -266,6 +266,12 @@ pub fn is_testnet() -> bool {
 	ChainTypes::Floonet == *param_ref
 }
 
+/// Are we for real?
+pub fn is_mainnet() -> bool {
```

Comment:
We added `is_mainnet()` back in? I thought we had removed this and replaced it with `!is_testnet()`.

The reason being we now have the majority of the tests running in an env that differs from mainnet.

Majority of the tests will return false here for `is_mainnet()` *and* for `is_testnet()`.